### DISCUSSION TITLE
Trigger welcome email webhook

### DISCUSSION THREAD

**Discussion by j2is** [swellstores/community#11] [Help] [answered]
What steps would I take to trigger a welcome email when a user is created manually by an admin in the backend of Swell?
The frontend triggers a welcome email anyway, it's just when creating a user in the backend.

**Comment by logeshswell** [comment, marked as answer]
If **Customer Welcome** notification is enabled, then the welcome email will be triggered for every new customer that you create manually with a default password set.

* You can navigate to **Settings** -> **Notifications** -> **Customers** in the Swell dashboard to confirm whether it is enabled.
* If the `password` field is not present while creating a customer, then it won't trigger the welcome email. But, you can override this password check by following the below steps:
1. Navigate to **Developer** -> **Console** in the Swell dashboard.
2. Send a GET request to `/:notifications/com.accounts.welcome.v2`.
3. From the received response, edit the `conditions` field or leave it empty as per your need and update it via PUT request.
